action/protocol/rewarding/reward_test.go

```diff
@@ -274,7 +280,8 @@ func TestProtocol_ClaimReward(t *testing.T) {
 		assert.Equal(t, big.NewInt(0), unclaimedBalance)
 		primAcc, err = accountutil.LoadAccount(sm, hash.BytesToHash160(claimActionCtx.Caller.Bytes()))
 		require.NoError(t, err)
-		assert.Equal(t, big.NewInt(1000010), primAcc.Balance)
+		initBalance = new(big.Int).Add(initBalance, big.NewInt(5))
+		assert.Equal(t, initBalance, primAcc.Balance)
```

Comment:
can simplify to 
`assert.Equal(t, initBalance.Add(big.NewInt(5)), primAcc.Balance)`